src/logic/prepare/provider/parameters.ts

```diff
@@ -0,0 +1,51 @@
+import type { IntegrationParameter } from '@superfaceai/ast';
+import { prepareProviderParameters } from '@superfaceai/ast';
+import inquirer from 'inquirer';
+
+export async function selectIntegrationParameters(provider: string): Promise<{
+  parameters: IntegrationParameter[];
+  values: Record<string, string>;
+}> {
+  const parameters: IntegrationParameter[] = [];
+  let exit = false;
+
+  while (!exit) {
+    const newParameter = await enterParameter(provider);
+    if (newParameter !== undefined) {
+      parameters.push(newParameter);
+    } else {
+      exit = true;
+    }
+  }
+
+  return {
+    parameters,
+    values: prepareProviderParameters(provider, parameters),
+  };
+}
+
+async function enterParameter(
+  provider: string
+): Promise<IntegrationParameter | undefined> {
+  const { name } = await inquirer.prompt<{ name: string }>({
+    name: 'name',
+    message: `Enter "name" of integration parameter for provider ${provider}, do not enter anything to end:`,
```

Comment:
I'd use something like

```suggestion
    message: `Add integration parameter (enter name; or leave empty to skip):`,
```